react/features/invite/components/add-people-dialog/native/AddPeopleDialog.js

```diff
@@ -405,8 +421,46 @@ class AddPeopleDialog extends AbstractAddPeopleDialog<Props, State> {
         );
     }
 
+    _renderInvitedItem: Object => ?React$Element<*>
+
+    /**
+     * Renders a single item in the invited {@code FlatList}.
+     *
+     * @param {Object} flatListItem - An item of the data array of the
+     * {@code FlatList}.
+     * @param {number} index - The index of the currently rendered item.
+     * @returns {?React$Element<*>}
+     */
+    _renderInvitedItem(flatListItem, index) {
+        const { item } = flatListItem;
+        const renderableItem = this._getRenderableItem(flatListItem);
+
+        return (
+            <TouchableOpacity onPress = { this._onPressItem(item) } >
```

Comment:
Question: should the search box be cleared when we select an item? It kinda makes sense so you can start typing another contact.

Reply:
May make sense, but I implemented it according to the figma prototype and that doesn't do that. But maybe it's just a limitation of that app. I will ask Tibi for sure